opensrp-giz-malawi/src/main/res/xml/captured_images_path.xml

```diff
@@ -1,4 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <paths>
     <external-path name="captured_photos" path="Android/data/org.smartregister.giz/files/Pictures" />
+    <external-path name="external_files" path="." />
+    <files-path name="root" path="." />
+    <external-path name="downloaded_files" path="Android/data/org.smartregister.giz/files/Pdf" />
```

Comment:
Is this need when file_paths.xml exists?

Reply:
Yes